src/pcm.c

```diff
@@ -28,12 +28,18 @@
 ** type and use SIZEOF_TRIBYTE instead of (tribyte).
 */
 
+#pragma pack(1)
```

Comment:
Humm, the idea of putting raw `#pragma` in the code smells funny to me. At the least this should be behind a macro, then it's a little more isolated if it has to be adapted for different compilers.

@SoapGentoo Thoughts?